tests/test_kernels.py

```diff
@@ -389,23 +381,9 @@ def test_ard_init_scalar(D):
 @pytest.mark.parametrize('N', [4, 7])
 @pytest.mark.parametrize('ard', [True, False, None])
 def test_ard_init_shapes(N, ard):
-    with pytest.raises(tf.errors.InvalidArgumentError):
-        k1 = gpflow.kernels.SquaredExponential(lengthscale=np.ones(2), ard=ard)
+    with pytest.raises(ValueError):
+        k1 = gpflow.kernels.SquaredExponential(lengthscale=np.ones(2))
         k1(rng.randn(N, 4))
-    with pytest.raises(tf.errors.InvalidArgumentError):
-        k2 = gpflow.kernels.SquaredExponential(lengthscale=np.ones(3), ard=ard)
+    with pytest.raises(ValueError):
+        k2 = gpflow.kernels.SquaredExponential(lengthscale=np.ones(3))
         k2(rng.randn(N, 2))
-
-
-@pytest.mark.parametrize('D', [4, 7])
-def test_ard_init_MLP(D):
```

Comment:
why did this disappear?

Reply:
This disappeared because the test is no longer valid - it was checking that if you passed in a scalar `weight_variances` with `ard=True` that it got expanded to an array. We now rely on the user explicitly specifying an array.